geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/test/scala/org/locationtech/geomesa/fs/storage/common/FileMetadataTest.scala

```diff
@@ -30,7 +30,7 @@ class FileMetadataTest extends Specification with AllExpectations {
   import scala.collection.JavaConverters._
 
   lazy val fc = FileContext.getFileContext(new Configuration())
-  val sft = SimpleFeatureTypes.createType("metadata", "name:String,dtg:Date,*geom:Point:srid=4326")
+  val sft = SimpleFeatureTypes.createType("metadata", "name:String,dtg:Date,*geom:Point:srid=4326;geomesa.user-data.prefix=desc,desc.name=姓名,desc.dtg=ひづけ,desc.geom=좌표")
```

Comment:
it would be good to add an explicit test that verifies that the user data is readable as expected.